libcanard/canard.c

```diff
@@ -66,6 +66,8 @@
 
 #define INITIAL_TOGGLE_STATE true
 
+#define CANARD_MTU_MAX CANARD_MTU_CAN_FD
```

Comment:
I think this actually should be in the header file (as I wrote earlier) because this is also useful for the API.